Pango-1.0.gir

```diff
@@ -4358,6 +4360,14 @@ for enumerating faces.</doc>
           </parameter>
         </parameters>
       </method>
+      <property name="item-type" transfer-ownership="none">
```

Comment:
Should really fix https://github.com/gtk-rs/gir/issues/1369 first before merging this. It would otherwise add a lot of noise.

Reply:
Its also missing Since: upstream

https://gitlab.gnome.org/GNOME/pango/-/merge_requests/624

Reply:
See my comment there

Reply:
This can land by the way, the properties are unrelated to the ListModel iface implementation as it doesn't require those. See https://github.com/gtk-rs/gtk4-rs/pull/1078/commits/b9cde4befdbd5cf698d5642654d77694b58e0580